packages/grafana-ui/src/components/uPlot/config/UPlotConfigBuilder.ts

```diff
@@ -238,7 +238,10 @@ export class UPlotConfigBuilder {
     );
 
     config.tzDate = this.tzDate;
-    config.padding = this.padding;
```

Comment:
regressed only 11 months ago :sweat_smile: https://github.com/grafana/grafana/pull/38881

the reason it breaks is cause we spread the default props with or reduced padding deeper: https://github.com/grafana/grafana/blob/6128cb60b4d38e49a069d81c492b1078e94300fb/packages/grafana-ui/src/components/uPlot/Plot.tsx#L68, so that started getting overwritten by `undefined`.

@dprokop maybe we should be merging those default props inside of getConfig() here instead? at least it'll all live in one place.

Reply:
Think it's a good idea, nothing more to say :D